CMSgov/onemac
# Oy2 22443 - update excel export for package view

Story: https://qmacbis.atlassian.net/browse/OY2-22443
Endpoint: See github-actions bot comment

### Details

Fix package view export to excel

### Changes

- Use new column header names
- Use package view onemac type lookup instead of change request type lookup

### Test Plan

1. Login as system admin
2. Navigate to package view
3. export spa list
4. verify spa list export has new header value "Initial Submission Date" and Type of "Medicaid SPA"
5. export waiver list
6. verify waiver list export has new header value "Initial Submission Date" and Type of "Temporary Extension"
